### ISSUE TITLE
Notebooks: notebook ui not working with api base path

### ISSUE THREAD

**Issue by DonCamillo72** [influxdata/ui#6664]
## About the bug

**Steps to reproduce:**
List the minimal actions needed to reproduce the behavior.

1. Build influx.ui with environment variables [API_BASE_PATH](https://github.com/influxdata/ui/blob/61852ce0f62c11f892d7c94d4d1c05ff6bea2f9a/src/shared/constants/index.ts#L88)="/influxdb/" and [BASE_PATH](https://github.com/influxdata/ui/blob/61852ce0f62c11f892d7c94d4d1c05ff6bea2f9a/src/shared/constants/index.ts#L87)="/influxdb/"
2. Start influxd with [assets path ](https://docs.influxdata.com/influxdb/v2.6/reference/config-options/?t=JSON#assets-path)of custom influx.ui build
3. Click on sidebar link "Notebooks" and
4. Click on "Create a Notebook" button

**Expected behavior:**

A new notebook is created.

**Actual behavior:**

Message "Failed to create Notebook, please try again" appears.
No notebook is created.

**Visual Proof:**

## About your environment

**Environment info:**

OS: Linux 5.4.0-144-generic x86_64
Version: InfluxDB v2.6.1

**Config:**

The configuration is set to default except for the assets-path option, see 'Steps to reproduce'. 
The assets-path is set to custom UI build result with environment variable API_BASE_PATH="/influxdb" and BASE_PATH="/influxdb/".

**Comment by zoesteinkamp** [comment]
Is this still an issue for you?

**Comment by DonCamillo72** [comment]
> Is this still an issue for you?

Yes, the notebook feature is still not working as described above.
Thanks for investigating the issue.
